ml/notebooks/netdata_anomaly_detection_deepdive.ipynb

```diff
@@ -0,0 +1,1705 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #!pip install netdata-pandas
</pre><p>After this, I would include: </p><pre class="ql-syntax" spellcheck="false">    "Once you have [netdata-pandas](https://github.com/netdata/netdata-pandas) installed, you can comment it back out and rerun the cell to clear the output."
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netdata/netdata/pull/12313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
did something similar in comment in the code cell so its as unmissable as possible